src/Core/Settings.h

```diff
@@ -438,6 +438,7 @@ struct Settings : public SettingsCollection<Settings>
     M(SettingUInt64, output_format_pretty_max_column_pad_width, 250, "Maximum width to pad all values in a column in Pretty formats.", 0) \
     M(SettingUInt64, output_format_pretty_max_value_width, 10000, "Maximum width of value to display in Pretty formats. If greater - it will be cut.", 0) \
     M(SettingBool, output_format_pretty_color, true, "Use ANSI escape sequences to paint colors in Pretty formats", 0) \
+    M(SettingString, output_format_pretty_grid_charset, "utf8", "Charset for printing grid borders. Available charsets: ascii, utf8 (default one).", 0) \
```

Comment:
Ok, but UTF-8 is written as `UTF-8` (with dash).

Why it is important?
Some tools (e.g. Perl) have both variants - UTF8 and UTF-8 that have different meanings.

See https://en.wikipedia.org/wiki/UTF-8 (official name and variants).

Also let's write them in capitals: ASCII, UTF-8.

Reply:
Made them in capitals.